docs/troubleshoot-collection.md

```diff
@@ -428,11 +429,11 @@ kube-prometheus-stack:
   prometheus:
     prometheusSpec:
       remoteWrite:
-        - url: http://$(METADATA_METRICS_SVC).$(NAMESPACE).svc.cluster.local.:9888/prometheus.metrics.state
+        - url: http://$(METADATA_METRICS_SVC).$(NAMESPACE).svc.cluster.local.:9888/prometheus.metrics
```

Comment:
Error above is about `http://sumologic-sumologic-remote-write-proxy.sumologic.svc.cluster.local:9888/prometheus.metrics.kubelet`